bamboo/bootstrap-integration-tests.sh

```diff
@@ -24,7 +24,8 @@ if [[ $USE_TERRAFORM_ZIPS == true ]]; then
 
   ## [MHS, 04/29/2021] fix cumulus_distribution package not released separately.
   sed -i "s/source[ ]*= \"..\/..\/tf-modules\/cumulus_distribution\"/source = \"https:\/\/github.com\/nasa\/cumulus\/releases\/download\/$VERSION_FLAG\/terraform-aws-cumulus.zip\/\/tf-modules\/cumulus_distribution\"/g" *.tf
-
+  # Update to use distribution module
```

Comment:
Where did this come from? Not saying it's wrong but I wouldn't expect code changes in the merge from the release branch to main.

Reply:
The update is to fix CI integration tests failure when being run against the distribution module with `USE_TERRAFORM_ZIPS=true` on the Release branch.
This is also mentioned in the Changed section of the Release Note.